packages/sdk/src/client.spec.ts

```diff
@@ -4,16 +4,24 @@ import { RpcParams, RpcResult, endpoints, getAddresses } from '@leather.io/rpc';
 
 import { createLeatherClient } from './client';
 
-vi.stubGlobal('LeatherProvider', {
-  request: () => Promise.resolve(),
-});
-
 describe('Leather SDK', () => {
+  vi.stubGlobal('LeatherProvider', {
+    request: () => Promise.resolve(),
+  });
   const client = createLeatherClient();
 
-  const clientMethods = Object.keys(client);
+  const methods = Object.keys(client);
   test('that it has a method for each endpoint', () => {
-    Object.values(endpoints).forEach(endpoint => clientMethods.includes(endpoint.method));
+    Object.values(endpoints).forEach(endpoint => methods.includes(endpoint.method));
+  });
+
+  // write test that ensures the onprovider call isn't called
+  test('onProviderNotFound is called if provider is available', () => {
```

Comment:
Finding the comment here and test confusing, but maybe something you are coming back to? Is this testing if it is or isn't called?

Reply:
sorry that was a comment for AI prompt 😅